monitoring/LOGGING.md

```diff
@@ -0,0 +1,31 @@
+## How to run monitoring services?
+It is simple!
+```bash
+git clone https://github.com/justmark0/DevOpsIU.git
+cd DevOpsIU/monitoring
+docker-compose up -d
+``` 
+And you are done!!!
+## How to access monitoring services?
+It is simple!
+Just run with 3 commands above and then open your browser and go to:
+* App itself - http://localhost:8000
+* Grafana - http://localhost:3000
+* Loki - http://localhost:9080/targets
+* Promtail - http://localhost:3100/ready
+
+And that is it!!! how simple was it, huh?
+
+Here is some awesome screenshots of what you can see in Grafana 💖🤤👉👈💖🤤👉👈💖🤤👉👈💖🤤👉👈💖🤤👉👈: 
+
+![Screenshot 2022-10-17 at 20 49 26](https://user-images.githubusercontent.com/54911879/196252866-dd807887-e326-4370-a01d-3af36e0b6037.jpg)
+
+
+And app itself!!:💥🔥💥🔥💥🔥💥🔥💥🔥💥🔥💥
+
+![Screenshot 2022-10-17 at 21 04 34](https://user-images.githubusercontent.com/54911879/196252852-85d91ecf-d8ce-42c8-97c4-bbb09a68851d.png)
+
+
+HERE IS AN INSANE PIC OF LOKI!!!!!! WOOOOOW😳😳😳😳😳😳😳😳😳😳😳
```

Comment:
![image](https://user-images.githubusercontent.com/49134679/196254257-0ba2d74f-3027-4aec-9bed-b53087150119.png)

Reply:
i mean it literally says promtail in the corner